README.md

```diff
@@ -83,6 +83,36 @@ sudo cp protoc-gen-gogoslick /usr/bin/
 
 Done
 
+## Running p2p Prometheus dashboards
+1. Start the node with `--p2p-prometheus-metrics` flag. This exposes a metrics collection at http://localhost:5001/debug/metrics/prometheus.
+2. Clone libp2p repository: `git clone https://github.com/libp2p/go-libp2p`
+3. `cd go-libp2p/dasboards/swarm` and under the 
+```json  
```

Comment:
might remove the `json` tag because the code is not json compatible (missing `], }` at the end)
Valid for L95. This will improve the readability in github